jobrunner/service.py

```diff
@@ -24,6 +24,9 @@
 
 def main():
     """Run the main run loop after starting the sync loop in a thread."""
+    threading.current_thread().name = "run "
```

Comment:
Did you mean to leave a trailing space here?

Reply:
Yes, so that it lines up with "sync" for the other thread. But worth a comment, will add.